preproccesing.py

```diff
@@ -0,0 +1,100 @@
+import json
+import multiprocessing
+import os
+from functools import partial
+from multiprocessing import Pool
+import argparse
+import cv2
+import numpy as np
+import open3d
+import configparser
+from imageio import imwrite
+
+
+def undistort(path, fx, fy, cx, cy, k1, k2, k3, k4, k5, k6, p1, p2):
+    image_ = cv2.imread(path, -1)
+    camera_matrix = np.array([[fx, 0, cx],
+                              [0, fy, cy],
+                              [0, 0, 1]])
+    undistorted = cv2.undistort(image_, camera_matrix, np.array([k1, k2, p1, p2, k3, k4, k5, k6]))
```

Comment:
instead of passing all k, you can once create list of k and pass it as parameter
(current function parameter list looks scary)